node/src/components/block_synchronizer/block_acquisition.rs

```diff
@@ -1002,35 +1006,36 @@ impl BlockAcquisitionState {
                         block_execution_results_or_chunk,
                         deploy_hashes,
                     ) {
-                    Ok(new_effects) => match new_effects {
+                    Ok((new_acquistion, acceptance)) => match new_acquistion {
```

Comment:
```suggestion
                    Ok((new_acquisition, acceptance)) => match new_acquisition {
```

Reply:
:scream: Fixed in [60de830](https://github.com/casper-network/casper-node/pull/4039/commits/60de830640564efa0db2de1200584f2cac31bbc6).